src/Services/login.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Users } from '../Types/Users';
+import { UsersModel } from '../Models/users.model';
+import bcrypt from 'bcrypt';
+
+export const checkLoginData = async (loginData: Users) => {
```

Comment:
add return type

Reply:
also I think that Users type should be renamed to User since it refers to only one User

Reply:
resolved